File: api/core/src/main/kotlin/data/UcenjeData.kt
package data

import domain.Oseba
import domain.Ucenje
import kotlinx.serialization.Serializable

@Serializable
data class UcenjeData(
    val ucenje: Ucenje,
    val oseba_refs: List<OsebaData> = listOf(),
    val ucenje_ucitelj_refs: List<UcenjeData> = listOf(),
)

File: app/src/app/core/services/api/models/ucenje-data.ts
/* tslint:disable */
/* eslint-disable */
import { OsebaData } from './oseba-data';
import { Ucenje } from './ucenje';
export interface UcenjeData {
  oseba_refs: Array<OsebaData>;
  ucenje: Ucenje;
  ucenje_ucitelj_refs: Array<UcenjeData>;
}
